src/lib/utilities.js

```diff
@@ -63,3 +63,30 @@ export const concatenateUint8Arrays = (...arrays) => {
   }, 0);
   return result;
 };
+
+export const arrayBufferToHex = buffer => {
+  if (buffer.buffer instanceof ArrayBuffer && buffer.byteLength !== undefined) {
+    throw new TypeError("Expected input to be an ArrayBuffer");
+  }
+  return Array.prototype.map
+    .call(new Uint8Array(buffer), x => `00${x.toString(16)}`.slice(-2))
+    .join("");
+};
+
+export const hexToArrayBuffer = hex => {
```

Comment:
This is only used in the test, i recommend we just get rid of this (see comment in test file)

Reply:
I need this func to use it also in the contract tests. 

For instance to validate that the lib works between the different sources (ios and android) what we will do is that they will encrypt some sort of data, then will give me the result o this encryption as a `hex` value since passing around ArrayBuffers is not working well for us. 

Then I will have this hex value and try to decrypt it to see if my decryption works as expected. In that case i will have to convert the hex to ArrayBuffer and therefore I will need something like this. 

What you think?

Reply:
still should be part of the tests and actually inlined as @Weetbix recommended

Reply:
so we will have this private function inside the tests to transform from hex to array buffer? 
but don't we want to test that this function actually does what it suppose to and therefore get tested? 
Adding a function to the tests that will implement some logic can make our test pass or fail unpredictably. For instance it can be that this private function has a bug and therefore the tests are passing when it should fail. 
I think it makes sense that we test this function before we use it in the tests. 

What you think?

Reply:
we just ask you to get rid of it and inline the result where needed.

In case you want to have something for tests - move it to tests scope, and create test nearby, it shouldn't be publicly available.